README.md

```diff
@@ -314,12 +314,17 @@ List of deprecated configuration parameters:
 - `output_fields` - A comma separated list of fields to include in output. Supported values are: `key`, `offset`, `timestamp` and `value`. Defaults to `value`.
 
 List of new configuration parameters:
-- `aws.access.key.id` - AWS Access Key ID for accessing S3 bucket. Mandatory.
-- `aws.secret.access.key` - AWS S3 Secret Access Key. Mandatory.
+- `aws.access.key.id` - AWS Access Key ID for accessing S3 bucket.
+- `aws.secret.access.key` - AWS S3 Secret Access Key.
 - `aws.s3.bucket.name` - - Name of an existing bucket for storing the records. Mandatory.
 - `aws.s3.endpoint` - The endpoint configuration (service endpoint & signing region) to be used for requests.
 - `aws.s3.prefix` - [Deprecated] Use `file.name.prefix` and `file.name.template` instead. The prefix that will be added to the file name in the bucket. Can be used for putting output files into a subdirectory.
 - `aws.s3.region` - Name of the region for the bucket used for storing the records. Defaults to `us-east-1`.
+- `aws.sts.role.arn` - AWS role ARN, for cross-account access role instead of `aws.access.key.id` and `aws.secret.access.key`
+- `aws.sts.role.external.id` - AWS ExternalId for cross-account access role
+- `aws.sts.role.session.name` - AWS session name for cross-account access role
+- `aws.sts.role.session.duration` - Session duration for cross-account access role in Seconds. Minimum value - 900. 
+- `aws.sts.config.endpoint` - AWS STS endpoint for cross-account access role
```

Comment:
Instead of the endpoint, can't we use regions? AFAIK, SDK will automatically determine the endpoint based on the region name.

Reply:
It turns out, if connector is not located inside AWS cloud it is faster to specify corresponding endpoint. Also from a documentation: "AWS STS recommends that you make calls to a Regional endpoint." https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html#id_credentials_temp_enable-regions_writing_code

Reply:
But I can add a default config value for a global endpoint, so that user had an option only specify 'S3 region'